source/popup.js

```diff
@@ -255,6 +255,7 @@ function sessionToButton(settings, session) {
 	if (settings.timer) {
 		object.lastModified = session.lastModified;
 	}
+        if (session.window !== undefined) object.open = settings.expand;	
```

Comment:
Could've surrounded the statement with curly braces...

But I should've had look at github notifications and merge this sooner.